js/ChannelFactory.js

```diff
@@ -1,141 +1,161 @@
+        //and (2) any configured resolver that does other lookup such as by name
+        resolvers = [function (schema) {
+            if (typeof schema === "object") {
+                return schema;
+            }
+        }];
+
+        if (config.resolver) {
+            resolvers.push(config.resolver);
+        }
+
+        /**
+         * Given a name of a schema iterate over the resolver array until you find a match.
+         * @param name
+         * @return {*}
+         */
+        this.resolveSchema = function (name) {
+            
+        	//The case we don't want to lookup and utilize a schema
+            if(name == 'default'){
+            	return {
+            	    "id": "schema/DefaultChannel"
+            	}
+            }
+            
+            var schema;
+            resolvers.some(function (res) {
+                schema = res(name);
+                return schema;
+            }, this);
+            return schema;
+        };
+
+        /**
+         * CoreApi instance used by this factory exposed.
+         * @type {CoreApi}
+         */
+        this.coreapi = new CoreApi({resolve: this.resolveSchema, validate: config.validate});
+
+        /**
+         * Add a resolver method to the internal array used to resolve schemas.
+         * @param newResolver - a function accepting (schemaName) that attempts to map that
+         * name to a schema object.
+         */
+        this.addResolver = function (newResolver) {
+            resolvers.push(newResolver);
+        };
+
+        /**
+         * Retrieve a Channel object based on the passed name, scope and bus.
+         * @param channelOrName - Either a direct channel schema instance, or a path to one that can be used to look it up with the resolver function.
+         * @param scope - the scope identifying the subscriber and used as the callback
+         * scope for subscriptions.
+         * @param busName - the optional bus to associate the channel with.
+         * @return {*}
+         */
+        this.get = function (channelOrName, scope, busName) {
+            var schema,
+                channel;
+
+            if (typeof (busName) === 'undefined') {
+                busName = 'global';
+            }
+
+            schema = this.resolveSchema(channelOrName);
+
+            log.debug("Getting channel " + schema.id + " on bus " + busName + " with scope " + scope);
+
+            channel = new Channel();
+            setProps(channel, schema, scope, this, busName);
+            return channel;
+        };
+        
+        /**
+         * Retrieve a default Channel object based on the passed scope and bus
+         * @param scope - the scope identifying the subscriber and used as the callback
+         * scope for subscriptions.
+         * @param busName - the optional bus to associate the channel with.
+         * @return {*}
+         */
+        
+        this.getChannelWithDefaultSchema = function (scope, busName){   

```

Comment:
The main `get` function should still suffice, because while they may not want to supply a schema, they still do need a channel name. So the first `channelOrName` arg should still be required - it just shouldn't actually need to resolve to a schema object. For pubsub, you'll always need to use a channel name in order to allow subscribers to get your message, but schemas should only be required if validation is desired.